fixtures/setup.json

```diff
@@ -17,6 +17,7 @@
       "commR": {"/":"bafk4ehzaqugph7z7yajugw6wz6nn7x3akr3e32wecwexllfgqndnjt3nsaya"},
       "commD": {"/":"bafk4chzavlx2s3zsomp6beohyevpbql477qyr7qnvgkvfwbbcqzfijo62arq"},
       "sectorNum": 3,
+      "proofType": 4,
```

Comment:
The proof type must be added to the SectorInfo on chain, so I've added it to the genesis config.